projects/plugins/crm/src/automation/commons/conditions/contacts/class-contact-status-changed.php

```diff
@@ -0,0 +1,190 @@
+<?php
+/**
+ * Jetpack CRM Automation Contact_Status_Changed condition.
+ *
+ * @package automattic/jetpack-crm
+ */
+
+namespace Automattic\Jetpack\CRM\Automation\Conditions;
+
+use Automattic\Jetpack\CRM\Automation\Automation_Exception;
+use Automattic\Jetpack\CRM\Automation\Automation_Logger;
+use Automattic\Jetpack\CRM\Automation\Base_Condition;
+
+/**
+ * Contact_Status_Changed condition class.
+ *
+ * @since $$next-version$$
+ */
+class Contact_Status_Changed extends Base_Condition {
+
+	/**
+	 * The Automation logger.
+	 *
+	 * @since $$next-version$$
+	 * @var Automation_Logger $logger The Automation logger.
+	 */
+	private $logger;
+
+	/**
+	 * All valid operators for this condition.
+	 *
+	 * @since $$next-version$$
+	 * @var string[] $valid_operators Valid operators.
+	 */
+	private $valid_operators = array(
+		'is',
+		'is_not',
+	);
+
+	/**
+	 * All valid attributes for this condition.
+	 *
+	 * @since $$next-version$$
+	 * @var string[] $valid_operators Valid attributes.
+	 */
+	private $valid_attributes = array(
+		'field',
+		'operator',
+		'value',
+	);
+
+	/**
+	 * Contact_Status_Changed constructor.
+	 *
+	 * @since $$next-version$$
+	 *
+	 * @param array $step_data The step data for the condition.
+	 */
+	public function __construct( array $step_data ) {
+		parent::__construct( $step_data );
+
+		$this->logger = Automation_Logger::instance();
+	}
+
+	/**
+	 * Executes the condition. If the condition is met, the value stored in the
+	 * attribute $condition_met is set to true; otherwise, it is set to false.
+	 *
+	 * @since $$next-version$$
+	 *
+	 * @param array $data The data this condition has to evaluate.
+	 * @return void
+	 *
+	 * @throws Automation_Exception If an invalid operator is encountered.
+	 */
+	public function execute( array $data ) {
+		if ( ! $this->is_valid_contact_status_changed_data( $data ) ) {
+			$this->logger->log( 'Invalid contact status changed data', $data );
+			$this->condition_met = false;
+			return;
+		}
+
+		$field    = $this->get_attributes()['field'];
```

Comment:
I just realized that for this Condition we don't need the field, because it is a specific condition for the field status.